tests/network/services.go

```diff
@@ -154,64 +148,55 @@ var _ = SIGDescribe("Services", func() {
 		})
 
 		Context("with a service matching the vmi exposed", func() {
-			var jobsToCleanup []func() error
+			var jobToCleanup func() error
 
 			BeforeEach(func() {
 				serviceName = "myservice"
 
 				service := buildServiceSpec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
 				_, err := virtClient.CoreV1().Services(inboundVMI.Namespace).Create(service)
 				Expect(err).ToNot(HaveOccurred())
-
-				// reset the job cleanup functions list
-				jobsToCleanup = nil
 			})
 
 			AfterEach(func() {
-				Expect(virtClient.CoreV1().Services(inboundVMI.Namespace).Delete(serviceName, &k8smetav1.DeleteOptions{})).To(Succeed())
+				Expect(virtClient.CoreV1().Services(inboundVMI.Namespace).Delete(serviceName, &k8smetav1.DeleteOptions{})).To(Succeed(), "cleaning up the k8sv1.Service entity should have succeeded.")
```

Comment:
Why aren't you using the new introduced method `cleanupService`here?

Reply:
I forgot this one existed; thanks for spotting.

Done.